catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -295,6 +295,7 @@
                             ddf.catalog.filter.impl,
                             ddf.catalog.history,
                             ddf.catalog.impl,
+                            ddf.catalog.impl.operations,
```

Comment:
I did. We may be able to clean up this bundle definition so it makes more sense, but as of right now this needed to be added as a private package or the bundle can't come up.